src/plugins/home/server/services/sample_data/data_sets/ecommerce/index.ts

```diff
@@ -55,13 +55,11 @@ export const ecommerceSpecProvider = function (): SampleDatasetSchema {
     darkPreviewImagePath: '/plugins/home/assets/sample_data_resources/ecommerce/dashboard_dark.png',
     hasNewThemeImages: true,
     overviewDashboard: DASHBOARD_ID,
-    getDataSourceIntegratedDashboard: appendDataSourceId(DASHBOARD_ID),
+    getDashboardWithPrefix: addPrefixTo(DASHBOARD_ID),
     appLinks: initialAppLinks,
     defaultIndex: DEFAULT_INDEX,
-    getDataSourceIntegratedDefaultIndex: appendDataSourceId(DEFAULT_INDEX),
+    getDataSourceIntegratedDefaultIndex: addPrefixTo(DEFAULT_INDEX),
     savedObjects: getSavedObjects(),
-    getDataSourceIntegratedSavedObjects: (dataSourceId?: string, dataSourceTitle?: string) =>
-      getSavedObjectsWithDataSource(getSavedObjects(), dataSourceId, dataSourceTitle),
```

Comment:
Is it save to delete this field?

Reply:
This method has been moved to `src/plugins/home/server/services/sample_data/data_sets/util.ts`, searching all OSD, no other places use it.